InventoryManagement/src/main/java/com/ccs/inventorymanagement/service/ClothingService.java

```diff
@@ -40,11 +41,18 @@ public Mono<Clothing> create(Clothing item) {
 
     @Override
     public Mono<Clothing> update(Clothing item) {
-        return null;
+        return clothingRepository.save(ClothingEntity.from(item)
```

Comment:
You should find the item first before saving it, if it doesn't exist then you would create it when an update was the intention.